inji-web/src/redux/reducers/commonReducer.ts

```diff
@@ -4,10 +4,12 @@ import {defaultLanguage} from "../../utils/i18n";
 
 const initialState = {
     language: storage.getItem(storage.SELECTED_LANGUAGE) ? storage.getItem(storage.SELECTED_LANGUAGE) : defaultLanguage,
+    vcExpiryTimes: 1
 }
 
 const CommonReducerAction: CommonReducerActionType = {
     STORE_LANGUAGE: 'STORE_LANGUAGE',
+    STORE_VC_EXPIRY_TIMES: 'STORE_VC_EXPIRY_TIMES'
```

Comment:
rename it to `STORE_VC_STORAGE_LIMIT_EXPIRY_IN_TIMES`

Reply:
Renamed it.